POSserver/server/models/job.py

```diff
@@ -1,5 +1,18 @@
 from django.db import models
+from .tag import Tag
+from .contractor import Contractor
+from uuid import uuid4
 
 
 class Job(models.Model):
-    pass
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    contractor_id = models.ForeignKey(Contractor, on_delete=models.CASCADE)
+    tag_id = models.ForeignKey(Tag, on_delete=models.CASCADE)
+    name = models.CharField(max_length=200)
+    notes = models.TextField(blank=True)
+    complete = models.BooleanField(default=False)
+    labor = models.DecimalField(null=True)
+    description = models.TextField(null=True)
+    created_at = models.DateTimeField(auto_now_add=True)
+    modified_at = models.DateTimeField(auto_now=True)
+    deadline = models.DateField(blank=True)
```

Comment:
I'm assuming here Client_job was replaced by Account_Job? what drove that change? Doesn't each client have just 1 account...inspite of having possibly many different engagements?

This was supposed to be on Zach's Account_job PR..but for whatever reason it won't let me do a code review there